accumulo1.7/src/main/conf/accumulo.properties

```diff
@@ -0,0 +1,44 @@
+# Copyright 2014 Cloudera, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Intentional?

Reply:
must be from before we switched to the "YCSB contributors" language.  not sure if copying it into a new directory is a creative act worthy of updating though.